src/EnergyPlus/PurchasedAirManager.cc

```diff
@@ -8,7 +8,7 @@
 // granted for itself and others acting on its behalf a paid-up, nonexclusive, irrevocable,
 // worldwide license in the Software to reproduce, distribute copies to the public, prepare
 // derivative works, and perform publicly and display publicly, and to permit others to do so.
-//
+//OutputRequiredToCO2SP
```

Comment:
Don't actually remove the whole line though.  Leave the `//` with no trailing spaces.